lib/perl/Genome/Ptero/Wrapper.pm

```diff
@@ -65,11 +66,13 @@ sub execute {
         $self->_run_command($cmd);
         return $cmd;
     } catch {
-        die $_;
-    } finally {
+        Carp::cluck($_);
         $self->_teardown_logging;
+        die $_;
```

Comment:
It might be prudent to capture $_ in a separate variable in case something in _teardown_logging overwrites it.